### PR TITLE
I18N Service Plugins - Improved error messaging 

### PR DESCRIPTION
**Pull request by Arvisix**
## Related Issue
https://github.com/webiny/webiny-js/issues/748

## Your solution
I added validation for valid url inside `packages/api-i18n/src/plugins/service.ts`

## How Has This Been Tested?
I don't know how this could be tested

### CODE REVIEW

File: packages/api-i18n/src/plugins/service.ts [hunk 1]
```diff
@@ -29,6 +29,14 @@ export default () => [
                 return localesCache;
             }
 
+            const urlRegEx = /^(https?:\/\/)((([a-z\d]([a-z\d-]*[a-z\d])*)\.)+[a-z]{2,}|((\d{1,3}\.){3}\d{1,3}))(\:\d+)?(\/[-a-z\d%_.~+]*)*(\?[;&a-z\d%_.~+=-]*)?(\#[-a-z\d_]*)?$/i;
+
+            if (!urlRegEx.test(process.env.I18N_API_URL)) {
```
Comment (doitadrian) [inline packages/api-i18n/src/plugins/service.ts]: Please use our own [validation library](https://github.com/webiny/webiny-js/tree/master/packages/validation) for checking URLs.

File: packages/api-i18n/src/plugins/service.ts [hunk 2]
```diff
@@ -29,6 +29,14 @@ export default () => [
...
+                throw Error(
+                    `I18N_API_URL env variable is not a valid URL. Make sure it's specified in the service configuration.`
```
Comment (doitadrian) [inline packages/api-i18n/src/plugins/service.ts]: I would just mention the exact file we are talking about here and that is the `serverless.yml`. This way users will know what exactly we meant, "service configuration" to some users might be confusing.

**Comment by Arvisix** [comment]
@doitadrian please check updated PR

File: packages/api-i18n/src/plugins/service.ts [hunk 3]
```diff
@@ -29,6 +30,18 @@ export default () => [
                 return localesCache;
             }
 
+            if (!process.env.I18N_API_URL) {
+                throw new Error(
+                    `I18N_API_URL env variable is missing. Make sure it's specified in the service 'serverless.yml' configuration.`
```
Comment (doitadrian) [inline packages/api-i18n/src/plugins/service.ts]:
```js
throw new Error("I18N_API_URL environment variable is missing. Please check the service configuration, located in the "api/serverless.yml" file.`);
```

Comment (doitadrian) [reply packages/api-i18n/src/plugins/service.ts]: I changed the message a bit, just to kinda sync it with the lower one.

File: packages/api-i18n/src/plugins/service.ts [hunk 4]
```diff
@@ -29,6 +30,18 @@ export default () => [
...
+                );
+            }
+
+            if (validation.validate(process.env.I18N_API_URL, 'url', { throw: false }) instanceof ValidationError) {
```
[the remark this answers is not in the document]
Comment (Arvisix) [reply packages/api-i18n/src/plugins/service.ts]: Oh, sorry, I though I changed it to `validateSync`

**Pull request merged**